lib/diagnostic.rb

```diff
@@ -105,15 +122,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+  include Carnivorous
+  def roar
+    p "ROAR!"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# Using mixins/composition allows developers to keep their
```

Comment:
The primary advantage is that classes can only inherit from one other class, while classes can include any number of modules.